src/app.js

```diff
@@ -63,10 +64,11 @@ server.get("/tweets/:username", (req, res) => {
 
   if (userTweets.length <= 10) {
     res.send([...userTweets].reverse());
-  } else {
-    const orderedUserTweets = userTweets.slice(-10).reverse();
-    res.send(orderedUserTweets);
```

Comment:
Geralmente a resposta final não fica inserida numa condicional

Reply:
Obs final: esqueceu de tratar esso quando query.page não é informado, ouseja é um Not A Number